tests/bootstrap.php

```diff
@@ -15,7 +15,7 @@ public function __construct() {
 
 		$this->tests_dir    = dirname( __FILE__ );
 		$this->plugin_dir   = __DIR__ . '/../../';
-		$this->wp_tests_dir = '/tmp/wordpress-tests-lib/';
+		$this->wp_tests_dir = !empty( getenv('WP_TESTS_DIR' ) ) ? getenv('WP_TESTS_DIR' ) : '/tmp/wordpress-tests-lib/';
```

Comment:
Minor tweaks - can you add a space between `!` and `empty`, space before `'WP_TESTS_DIR'`?

Reply:
This is now fix with the most recent commit. Thanks!